examples/femnist.py

```diff
@@ -27,6 +28,8 @@
 parser.add_argument("--num_pixel", type=int, default=28)
 parser.add_argument("--model", type=str, default="CNN")
 
+## algorithm
+parser.add_argument("--fed", type=str, default="Federated()")  ## Federated(), ICEADMM(), IIADMM()

```

Comment:
Should we come up with a better name for the argument? `--federation_type`? Can we drop `()` from the argument values?

Reply:
Reflected.